### PR TITLE
Add Component Input Fields

### LINKED ISSUE #4: [ MISC ] Components - Input Fields
There are four main input fields. See the [figma](https://www.figma.com/file/N16keN1mV1QDnorLB1GC6C/Plant-it-Again-(PIA)-Program-Manager?type=design&node-id=1034-6627&mode=design&t=7E2dgN76Hat7Zc6g-0) for details on what these components look like. I've also added some screenshots below for additional reference.

- Text input
- Calendar Input
- Checkboxes
- Radio Buttons

This contains the majority of the input components.

Screenshot 2024-01-10 at 12 40 43 AM

The calendar input should open some calendar view like this and users should be able to select a date.

Screenshot 2024-01-10 at 12 41 05 AM

Screenshot 2024-01-10 at 12 41 32 AM

### PR DESCRIPTION
**Pull request by aaronchan32**
## Tracking Info

Resolves #4 

## Changes
- Added global stylesheet, _app.jsx, and tailwind config to finish setting up tailwind
- Added Textfield, Checkbox, and Radio component
- Installed shadcn for Calendar and Popover components

## Testing

Created small demo on home page (index.tsx) demonstrating how to use each input component. Clicking the submit button will log out input values into the console.

## Confirmation of Change

image

### CODE REVIEW

**Comment by aaronchan32** [comment]
Converted to PR draft to improve multiple input support

File: frontend/src/pages/index.tsx
Comment (adhi0331) [inline frontend/src/pages/index.tsx:8]: Lets remove these components from the index page before merging since we're not going to need it here.

**Review by adhi0331 — APPROVED** [review]
LGTM!!! 🚀 (If you have the bandwidth it would be cool if their is a way for the other checkbox to revert back to being a checkbox if the field is empty).

**Review by andrewzpu — APPROVED** [review]
Overall, seems good to me. Some of the changes overlap with my PR #33 , including the button component and global CSS. We may have to go back through later and reconcile some of the differences (ie tailwind, hsl vs rgb, etc).

**Pull request merged**
